sample/AutoPyBind11Test.ipynb

```diff
@@ -165,7 +150,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "visualize_binding_code(BINDING_CODE_DIR)"
+    "visualize_binding_code(os.path.join(binding_DEST, \"%s.cpp\" % cmake_library_name))"
```

Comment:
@johnwparent Can you confirm that this change is OK?

Reply:
Using MSVC this path won't work (which is why it is the way it is) but for the binder instance this change works. Was hoping to make this repo functionality for a local checkout but if it's causing errors w/ Binder then this change is fine.

Reply:
I was under the assumption that `os.path.join` was for that exact purpose? Or do you mean the fact that we're referring to the `.` directory here?

Reply:
Right, the `binding_DEST` variable refers to the root of the build repo. In MSVC the build output is then nested under a directory specifying the build configuration. The separator is not the issue here, rather the directory structure produced by MSVC.

Reply:
Good catch. Probably needs to be reverted then.

Reply:
See altered f5e945e38a5c50c3be4fdf1a3e4f85e082f3171c